server/migrate.psql

```diff
@@ -252,6 +252,7 @@ CREATE TABLE public.packet (
     allowcharge boolean,
     contractorerror boolean,
     highvoltageenable boolean
+    lap boolean
```

Comment:
I think you need a comma on the line above

Reply:
Got you